### PR TITLE
Fix Typo: Use 'Proof Size' Instead of 'Proof Time'

### PR DESCRIPTION
**Pull request by pgherveou**

### CODE REVIEW

File: crates/env/src/call/call_builder.rs [hunk 1]
```diff
@@ -106,8 +106,8 @@ where
 
     /// Returns the chosen proof time limit for the called contract execution.
     #[inline]
-    pub fn proof_time_limit(&self) -> u64 {
-        self.call_type.proof_time_limit
+    pub fn proof_size_limit(&self) -> u64 {
```
Comment (ascjones) [inline crates/env/src/call/call_builder.rs:109]: Code comment needs updating too

**Review by ascjones — APPROVED** [review]
Should have noticed this when I was adding this API :see_no_evil:

**Comment by SkymanOne** [comment]
Why do you need to bump ink version? We can release it as part of stable 5.0.0 release

**Comment by ascjones** [comment]
Can we have a more descriptive title for this PR?

**Review by smiasojed — APPROVED** [review]

**Review by cmichi — APPROVED** [review]

**Pull request merged**